build.zig

```diff
@@ -52,6 +69,7 @@ pub fn build(b: *std.Build) void {
         .strip = if (debug_release) false else null,
     });
     l.link(&exe.root_module, false, .{});
+    exe.linkLibC();
```

Comment:
If we're gonna link libc, maybe we should support MUSL builds first.

At the very least we should advertise what libc ABI we're using in release binary names.

Reply:
musl sounds good to me